models/search_result.coffee

```diff
@@ -38,6 +38,8 @@ module.exports = class SearchResult extends Backbone.Model
       "/show/#{@get('id')}"
     else if @get('model') is 'sale'
       "/auction/#{@get('id')}"
+    else if @get('model') is 'city'
+      "/shows/#{@get('id')}"
```

Comment:
Great spot! This is fixed in this [Gravity PR](https://github.com/artsy/gravity/pull/10599) so let's wait for that to get deployed.